index.html

```diff
@@ -8,41 +9,343 @@
+                    you like what you see and have a project you need coded,
                     don’t hestiate to contact me.<br>
-                    <button type="button" class="connect">Lets Connect!
-                    </button>
-                </p>     
+                    <!-- <button type="button" class="connect">Lets Connect!
+                    </button> -->
+                </p>
                 <ul class="social-media-icons">
-                    <li class="social-media"><a href="https://twitter.com/" ><img src="./icons/Social_icon1.png"  alt="Twitter" ></a></li> 
-                    <li class="social-media"><a href="https://www.linkedin.com/" ><img src="./icons/Social_icon2.png"     alt="Linkedin" ></a></li> 
-                    <li class="social-media"><a  href="https://medium.com/" ><img src="./icons/Social_icon3.png"      alt="Medium" ></a></li> 
-                    <li class="social-media"><a href="https://angel.co/" ><img src="./icons/Social_icon5.png"      alt="angellist"></a></li>
-              </ul>          
+                    <li class="social-media"><a href="https://twitter.com/"><img src="./icons/Social_icon1.png"
+                                alt="Twitter"></a></li>
+                    <li class="social-media"><a href="https://www.linkedin.com/"><img src="./icons/Social_icon2.png"
+                                alt="Linkedin"></a></li>
+                    <li class="social-media"><a href="https://medium.com/"><img src="./icons/Social_icon3.png"
+                                alt="Medium"></a></li>
+                    <li class="social-media"><a href="https://angel.co/"><img src="./icons/Social_icon5.png"
+                                alt="angellist"></a></li>
+                </ul>
             </section>
         </div>
     </div>
+    <div class="container2">
+        <a href="#main-heading"><img class="arrow" src="./icons/down-arrow.svg" alt="Arrow"></a>
+        <h2 id="main-heading">My Recent <br>WORKS</h2>
+        <img id="indicator" src="./icons/Indicator.svg" alt="indicator">
+    </div>
+
+    <div class="card-holder">
+
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
+
+            </div>
+        </div>
```

Comment:
Your Design of the project card it's a bit large than the one in the Figma design, I suggest removing the white space and it should be the same as the Figma design 

- [ ] Figma design
![image](https://user-images.githubusercontent.com/61790587/164761510-b9db0caa-792c-4f25-bae8-c084a4105202.png)

- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164761545-d9582909-2c9f-4dc7-87de-43795c8da03d.png)